### DISCUSSION TITLE
Set MAC address yes or no?

### DISCUSSION THREAD

**Discussion by martin8310** [syssi/esphome-atorch-dl24#18] [Q&A] [answered]
Hello, 
Does the MAC address have to be adjusted? 

esp32-example.yaml file 
"dl24_mac_address: 41:B8:12:0A:A8:37"

Regards Martin

**Comment by syssi** [comment, marked as answer]
Yes. You have to use the MAC address of your device here.

**Reply by martin8310** [reply]
Thanks, do i need to adjust anything else?

**Reply by syssi** [reply]
No. That's all!

**Reply by martin8310** [reply]
Thank you.
